docs/docsite/rst/core_index.rst

```diff
@@ -36,6 +36,7 @@ This documentation covers the version of ``ansible-core`` noted in the upper lef
    :caption: Ansible getting started
 
    getting_started/index
+   getting_started_ee/index
```

Comment:
```suggestion
```

Reply:
LGTM, feel free to commit the suggestion